atom/browser/api/atom_api_protocol.h

```diff
@@ -28,6 +28,7 @@ namespace atom {
 
 namespace api {
 
+const void* DisableProtocolInterceptFlagKey();
```

Comment:
It would be better to declare the key in `AtomURLRequestJobFactory` and perform the check there. Should avoid the dependency of api class methods if possible.

Reply:
:+1: